pydicom/util/fixes.py

```diff
@@ -0,0 +1,128 @@
+"""Compatibility fixes for older version of python."""
+
+# Authors : Guillaume Lemaitre <g.lemaitre58@mgail.com>
+# License : MIT
+
```

Comment:
I guess you took this code from somewhere - in this case it would sense to mention where it is taken from (from Python? which version?)
Also, I think there has been some agreement about the copyright notice, though that may be still in discussion.